### ISSUE TITLE
Option to throttle throughput

### ISSUE THREAD

**Issue by ghost** [kzahel/jstorrent-legacy-app#58]
I'd like to see a feature to throttle the throughput of the up or down speeds.  Also would be cool to set up a temporary throttle, or even scheduled throttle similar to Transmission.

**Comment by kzahel** [comment]
Temporary throttle button? What does that do? pause downloads for 10 minutes?

**Comment by ghost** [comment]
Instead of full throughput (maximum bandwidth possible), cut the bandwidth
used by all torrents down to a set maximum so as not to interrupt regular
browsing experience.  That way you can still seed or download without
having to reestablish peers.

**Comment by beornmorder** [comment]
Agreed. A simple set download/upload speed would be an amazing first step.

**Comment by kzahel** [comment]
The legacy Chrome App is no longer maintained. I've rebuilt JSTorrent from scratch — the new version has upload and download speed limits built in. Get it at **[jstorrent.com](https://jstorrent.com)**

If you run into any issues, please leave feedback!
